spec/features/draft_taxons_spec.rb

```diff
@@ -86,6 +86,9 @@ def given_there_is_a_draft_taxon
   end
 
   def when_i_visit_the_taxon_page
+    stub_request(:get, "https://email-alert-api.test.gov.uk/subscriber-lists")
```

Comment:
I think this stub should probably go into the `given_there_are_taxons` method, that's where other requests that are made during page load are defined. This might apply to the other instances of this line in other spec files.